### PR TITLE
Revise copy

### PR DESCRIPTION
**Pull request by robertu7**
* Revise airdrop and pre-order copy;
* Add sign-in success page;
* Auto-refresh order details every 10s;

### CODE REVIEW

File: src/components/Buttons/SignInWithMatters/index.tsx [hunk 1]
```diff
@@ -43,14 +52,25 @@ export const SignInWithMatters = () => {
...
-      right={loading ? <IconSpinner /> : <IconArrowRight />}
-      htmlHref={env.mattersLoginURL}
-      htmlTarget="_blank"
-      onClick={signIn}
-      disabled={loading || !!error}
-    />
+    <>
+      <CardButton
+        title={
+          locale === Lang.en ? "Sign In with Matters" : "請登入 Matters 帳戶"
```
Comment (devformatters) [inline src/components/Buttons/SignInWithMatters/index.tsx:58]: Beryl uses `Matters.news` to specify the web app product.

File: src/components/Dialogs/PreOrderDialog/CompletedContent/index.tsx [hunk 2]
```diff
@@ -41,7 +51,7 @@ const CompletedContent: React.FC<CompletedContentProps> = ({
       </Dialog.Content>
 
       <Dialog.CTAButton color="blackLight" onClick={closeDialog}>
-        完成
+        {locale === Lang.en ? "OK" : "完成"}
```
Comment (devformatters) [inline src/components/Dialogs/PreOrderDialog/CompletedContent/index.tsx:54]: I found it is`Complete` in doc.

File: src/pages/404/index.tsx [hunk 3]
```diff
@@ -27,17 +27,14 @@ const NotFoundPage: React.FC<PageProps> = ({
...
-              <div className={styles.content}>
-                <h1 className={styles.title}>Not Found</h1>
-
-                <p className={styles.intro}>
-                  You just hit a route that doesn&#39;t exist... the sadness.
-                </p>
-              </div>
+              <MessageBox
+                title="Not Found"
+                description="You just hit a route that doesn&#39;t exist... the sadness."
```
Comment (devformatters) [inline src/pages/404/index.tsx:36]: Sadness. 😉

File: src/views/Homepage/CharacterIntro/index.tsx [hunk 4]
```diff
@@ -46,7 +46,7 @@ const CharacterIntro = () => {
               <p>
                 {locale === Lang.en
                   ? `We have created 6 basic characters for voyagers, namely, Gena, Lesya, Sasha, Valya, Yury, and Zhenya. Meanwhile, we have generated xxx(number) Traveloggers with character designs based on 6 aspects: value orientation, philosophical proposition, creative temperament, way of thinking, creative habits, and personality. Each Traveloggers is unique with its own creator attributes.`
-                  : `我們為遠航者創造了6個基礎人物，分別是 Sasha（薩沙）、Zhenya（振亞）、Valya（瓦拉）、Gena（根那）、Lesya（列峽）、Yury（尤里）。同時我們根據創作者的價值取向，哲學主張，創作者的創作氣質，思考方式，創作者的創作習慣和個性等6個維度，提取關鍵詞進行設計， 生成了xxx個CryptoMatties。每一個 CryptoMatties 都有專屬的創作者屬性，都是獨一無二的。`}
+                  : `我們為遠航者創造了6個基礎人物，分別是 Sasha（薩沙）、Zhenya（振亞）、Valya（瓦拉）、Gena（根那）、Lesya（列峽）、Yury（尤里）。同時我們根據創作者的價值取向，哲學主張，創作者的創作氣質，思考方式，創作者的創作習慣和個性等6個維度，提取關鍵詞進行設計， 生成了 xxx 個 CryptoMatties。每一個 CryptoMatties 都有專屬的創作者屬性，都是獨一無二的。`}
```
Comment (devformatters) [inline src/views/Homepage/CharacterIntro/index.tsx:49]: There's a newer version of it. Please check.

**Review by devformatters — CHANGES_REQUESTED** [review]

File: src/components/Buttons/SignInWithMatters/index.tsx (hunk 1 above)
Comment (robertu7) [reply src/components/Buttons/SignInWithMatters/index.tsx:58]: Yes. I think it's fine since the dialog description has already mentioned Matters.news.

Comment (devformatters) [reply src/components/Buttons/SignInWithMatters/index.tsx:58]: Got it. 👌🏿

File: src/views/Homepage/CharacterIntro/index.tsx [hunk 5]
```diff
@@ -45,8 +45,8 @@ const CharacterIntro = () => {
...
                 {locale === Lang.en
-                  ? `We have created 6 basic characters for voyagers, namely, Gena, Lesya, Sasha, Valya, Yury, and Zhenya. Meanwhile, we have generated xxx(number) Traveloggers with character designs based on 6 aspects: value orientation, philosophical proposition, creative temperament, way of thinking, creative habits, and personality. Each Traveloggers is unique with its own creator attributes.`
-                  : `我們為遠航者創造了6個基礎人物，分別是 Sasha（薩沙）、Zhenya（振亞）、Valya（瓦拉）、Gena（根那）、Lesya（列峽）、Yury（尤里）。同時我們根據創作者的價值取向，哲學主張，創作者的創作氣質，思考方式，創作者的創作習慣和個性等6個維度，提取關鍵詞進行設計， 生成了xxx個CryptoMatties。每一個 CryptoMatties 都有專屬的創作者屬性，都是獨一無二的。`}
+                  ? `Traveloggers have six basic characters, namely, Gena, Lesya, Sasha, Valya, Yury, and Zhenya. 1500 avatars were generated based on the combination of different accessories: value position, philosophical standpoint, way of thinking, creative habits, personality, etc. Each Traveloggers is unique with its own creator attributes.`
+                  : `Traveloggers 有6個基礎人物，分別是  Gena、Lesya、Sasha、Valya、Yury、Zhenya。我們根據創作者的價值取向、哲學主張、創作者的思考方式、創作習慣和個性等多個維度，提取關鍵元素組合，創造了1500 個 Traveloggers 的化身，每一個都獨一無二。`}
```
Comment (devformatters) [inline src/views/Homepage/CharacterIntro/index.tsx:49]: Spacing issue: `有6個` and `創造了1500 個`

File: src/views/Homepage/CharacterIntro/index.tsx [hunk 6]
```diff
@@ -58,8 +58,8 @@ const CharacterIntro = () => {
               <img src="/images/char1.png" />
               <p>
                 {locale === Lang.en
-                  ? "1 Character + 2 Accessories: Born of Matties"
-                  : "一個人物＋兩個配件：Matties 的誕生"}
+                  ? "1 Character + 2 Accessories: Birth of Traveloggers"
+                  : "一個人物＋兩個配件：Traveloggers 的誕"}
```
Comment (devformatters) [inline src/views/Homepage/CharacterIntro/index.tsx:62]: Lost `生`.

**Review by devformatters — APPROVED** [review]
Go. 👌🏿

**Review by guoliu — APPROVED** [review]

**Pull request merged**